README.md

```diff
@@ -84,9 +75,33 @@ CFLAGS="-I$GETTEXT/include -I$OPENSSL/include" LDFLAGS=-L$GETTEXT/lib ./configur
 
 You can just run `make` as usual after these steps.
 
+5. Install macOS with Homebrew
+------------------------------
 
-----------
-5. LICENSE
+Install the following homebrew packages:
+
+  `brew install axel`
+
```

Comment:
I think this section just needs the first two lines. If you're just installing axel with homebrew you don't need any of the instructions below - those are for building from source.

Please remove lines 85 - 93.